src/storage/conversion/RdfToQuadConverter.ts

```diff
@@ -23,7 +25,14 @@ export class RdfToQuadConverter extends BaseTypedRepresentationConverter {
     const rawQuads = rdfParser.parse(representation.data, {
       contentType: representation.metadata.contentType!,
       baseIRI: identifier.path,
-    });
+    })
+      // This works only for those cases where the data stream has been completely read before accessing the metadata.
+      // Eg. the PATCH operation, which is the main case why we store the prefixes in metadata here if there are any.
+      // See also https://github.com/CommunitySolidServer/CommunitySolidServer/issues/126
+      .on('prefix', (prefix, iri: NamedNode): void => {
+        representation.metadata.addQuad(iri.value, PREFERRED_PREFIX_TERM, prefix, SOLID_META.terms.ResponseMetadata);
+      });
```

Comment:
We might be able to get around this by forcing the converter to buffer some chunks already at this point, since prefixes are usually found at the beginning of a file.

But this would probably require very custom stream solutions so let's not go there until really needed.